grails-app/views/product/edit.gsp

```diff
@@ -18,6 +18,10 @@
         #category_id_chosen {
             width: 100% !important;
         }
+        .handling-icon-2x > i > svg {
```

Comment:
This is fine, but why can't we just add fa-2x to the icon attribute?

Reply:
You're right. I thought that it has to be with 'fa' class to be working and this is just font-size: 2em. I'll get rid of this class with my next changes.

Reply:
But instead of on icon, it should be on the outer element that is wrapping <g:renderHandlingIcons ... />